cmd/edv-rest/startcmd/start.go

```diff
@@ -83,17 +94,27 @@ type edvParameters struct {
 	databaseURL    string
 	databasePrefix string
 	logLevel       string
+	tlsConfig      *tlsConfig
+}
+
+type tlsConfig struct {
+	certFile string
+	keyFile  string
 }
 
 type server interface {
-	ListenAndServe(host string, router http.Handler) error
+	ListenAndServe(host, certFile, keyFile string, router http.Handler) error
 }
 
 // HTTPServer represents an actual HTTP server implementation.
 type HTTPServer struct{}
 
 // ListenAndServe starts the server using the standard Go HTTP server implementation.
-func (s *HTTPServer) ListenAndServe(host string, router http.Handler) error {
+func (s *HTTPServer) ListenAndServe(host, certFile, keyFile string, router http.Handler) error {
```

Comment:
given that this is an EDV, I would force to use httpS and, again, using TLS ending for function name to communicate and emphasize that

Reply:
+1 for forcing https

Reply:
You need to force https when it's configure with TLS if not use without tls

Reply:
@fqutishat 

@DRK3 mentioned something about this breaking in sandbox/AWS. Can you elaborate how it would break?

Reply:
we use proxy for https it's mean all application are http behind this proxy. The proxy is https.

Reply:
https://github.com/trustbloc/edge-sandbox/blob/master/test/bdd/fixtures/demo/docker-compose-third-party.yml#L102